Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -718,7 +739,7 @@ internal async Task<bool> IsSafeToProceed( EventCallback<CancellableRowChange<TI
 
         #region Filtering
 
-        protected void HandleSortColumn( DataGridColumn<TItem> column, bool changeSortDirection )
+        protected void HandleSortColumn( DataGridColumn<TItem> column, bool changeSortDirection, SortDirection? sortDirection = null )

```

Comment:
There's an async execution that's been added. Missing Task here?

Reply:
Async exec is inside of `InvokeAsync` so it should be good. But I can make it better.